File: Practice/30 Days of Code/Day 6 Let's Review/Day 6 Let's Review.php
<?php
$_fp = fopen("php://stdin", "r");
/* Enter your code here. Read input from STDIN. Print output to STDOUT */
fscanf($_fp, "%d\n", $n);
for($t=0; $t<$n; $t++) {
    fscanf($_fp, "%s\n", $str);
    for($i=0; $i<strlen($str); $i+=2) echo $str[$i];
    echo ' ';
    for($i=1; $i<strlen($str); $i+=2) echo $str[$i];
    echo "\n";
}
?>

File: Practice/30 Days of Code/Day 6 Let's Review/Day 6 Let's Review.java
import java.io.*;
import java.util.*;

public class Solution {

    public static void main(String[] args) {
        /* Enter your code here. Read input from STDIN. Print output to STDOUT. Your class should be named Solution. */
        Scanner in = new Scanner(System.in);
        int TT = in.nextInt();
        in.nextLine();
        for(int t=0; t<TT; t++) {
            char[] chs = in.nextLine().toCharArray();
            for(int i=0; i<chs.length; i+=2) {
                System.out.print(chs[i]);
            }
            System.out.print(" ");
            for(int i=1; i<chs.length; i+=2) {
                System.out.print(chs[i]);
            }
            System.out.println();
        }
    }
}